src/actions/index.js

```diff
@@ -0,0 +1,14 @@
+const addBook = (book) => ({
+  type: 'CREATE_BOOK',
+  payload: book,
+});
+
+const removeBook = (book) => ({
+  type: 'REMOVE_BOOK',
+  payload: book,
+});
+
+export default {
+  addBook,
+  removeBook,
+};
```

Comment:
You can only use **default export** for a file. 
However, where there are individual functions in a file, you use **named export** : putting the export in front of each function. Alternatively, at the end of the file, you export the functions like so:  export { addBook, removeBook } without the word **default**  

[Read this](https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910) for clarifications